samples/luis-bot-es6/app.js

```diff
@@ -49,90 +49,91 @@ adapter.use(state);
 
 // Listen for incoming requests
 server.post('/api/messages', (req, res) => {
+
     // Route received request to adapter for processing
     adapter.processActivity(req, res, async context => {
         if (context.activity.type === 'conversationUpdate' && context.activity.membersAdded[0].name === 'Bot') {
             // Welcome message here, bot will message you first
-            await context.sendActivity(`Hi! I'm a simple reminder bot. I can help add reminders, delete and show them.`);
+            return await context.sendActivity(`Hi! I'm a simple reminder bot. I can help add reminders, delete and show them.`);
         } else if (context.activity.type === 'message') {
+
+            // Handle a message from the user
             const utterance = (context.activity.text || '').trim().toLowerCase();
 
             // Create dialog context
             const dc = dialogs.createContext(context, state);
```

Comment:
>const dc = dialogs.createContext(context, state); [](start = 12, length = 49)

This isn't being setup correctly.  Your not actually getting the state object and its kind of working because you're using JS. TypeScript would have given an error.  Look at the dialog sample for how to properly get the state.